src/tools/periph/mkfatimage16.c

```diff
@@ -467,7 +467,7 @@ int main(int argc, char *argv[])
   bpb->num_heads = heads;
 
   if (!bootsect_file) {       // Assume writing for Dosemu / FreeDOS so v4 BPB
-    memcpy(buffer + 0x03, "DOSEMU10", 8);
+    memcpy(buffer + 0x03, "IBM  3.3", 8);
```

Comment:
Hmm, would it be possible to do all your
fancy guess-work based on oemname too?

Reply:
Just adding a couple of links here about significance of OEMname so they don't get lost:
https://web.archive.org/web/20170914204130/http://seasip.info/Misc/oemid.html
https://web.archive.org/web/20170909162834/http://jdebp.eu/FGA/volume-boot-block-oem-name-field.html

Reply:
This conversation seems to have been resolved.
Do your new findings change anything here?

Reply:
I think we are okay as is because nothing seems broken, though according to the FGA link, "IBM  2.0" would have been the optimum choice.